webapp/src/js/controllers/contacts-content.js

```diff
@@ -122,7 +124,9 @@ angular.module('inboxControllers').controller('ContactsContentCtrl',
     };
 
     // exposed solely for testing purposes
-    this.setupPromise = $q.resolve().then(function() {
+    this.setupPromise = $scope.getUserHomePlaceSummary().then(function(homePlace) {
```

Comment:
I think this is an unnecessary database lookup -- we only need the id of the facility here which we have in this class already on line 49 in `function getHomePlaceId`.